syntax = "proto3";

package freefire;

message PlayerInfo {
  string uid = 1;
  string server = 2;
  bytes account_info = 3;
  string PlayerNickname = 4;
  int32 Likes = 5;
  string Avatar = 6;
}

message LikeRequest {
  string target_uid = 1;
  string sender_uid = 2;
  string server = 3;
  int32 id = 4;
}

message LikeResponse {
  bool success = 1;
  string message = 2;
  int32 id = 3;
}

message AccountInfo {
  int32 UID = 1;
  string PlayerNickname = 2;
  int32 Likes = 3;
  int32 Level = 4;
  string Avatar = 5;
  string Region = 6;
  int64 LastLoginTime = 7;
}